Resources/Prototypes/Entities/Buildings/Doors/airlock_access.yml

```diff
@@ -0,0 +1,99 @@
+# Current access: "Maintenance", "Cargo", "Engineering", "Medical", "Research", "Security", "Service", "Captain", "Command", "External"
+- type: entity
+  parent: Airlock
+  id: AirlockLocked
+  suffix: Locked
+  components:
+  - type: AccessReader
+    necessary: ["Maintenance"]
```

Comment:
Why is maintenance the access level for this "plain" airlock?

Reply:
I thought we were going to use it as a placeholder for maint doors